test/parallel/test-url.js

```diff
@@ -1576,3 +1576,1347 @@ for (var i = 0; i < throws.length; i++) {
 };
 assert(url.format('') === '');
 assert(url.format({}) === '');
+
+
+var whatWgTests = [
```

Comment:
Maybe add a comment showing where these come from and explaining that the "nodejs" entry is not necessarily reflective of where we'd like to be, but instead of where we are.